.github/workflows/integration-test.yml

```
@@ -0,0 +1,141 @@
+            run: '^TestEngine'
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Generate Secrets Environment
+        run: ./github/scripts/setup/generate-secrets.sh
+        env:
+          NAME: ${{ matrix.integration.name }}
+          ENV_FILE: ${{ env.GITHUB_WORKSPACE }}/.env.secrets
+          SCRIPT_PATHS: ${{ matrix.integration.setup_scripts }}
```

actionlint (1.7.4)

object, array, and null values should not be evaluated in template with ${{ }} but evaluating the value of type array<string>

(expression)